### ISSUE TITLE
ENMI_ONE_ENUM_VALUE false positive

### ISSUE THREAD

**Issue by vsTianhao** [spotbugs/spotbugs#3114]
Very failed detection, it doesn’t even recognize basic method rewriting
``` java
// A ENMI_ONE_ENUM_VALUE 
enum A {
	aa(1) {
		@Override
		void impl() {
		}
	},
	bb(1) {
		@Override
		void impl() {
		}
	};
	A(int p) {}
	abstract void impl();
}
```

This kind of code is by no means uncommon, please refer to [CaseFormat](https://github.com/google/guava/blob/master/guava/src/com/google/common/base/CaseFormat.java), [LongMath](https://github.com/google/guava/blob/e5b58e2fc8561192338dfe0c68c0bce27066a6aa/guava/src/com/google/common/math/LongMath.java#L1090C16-L1090C34)

**Comment by gtoison** [comment]
Hello, this a bug reported by fb-contrib: https://github.com/mebigfatguy/fb-contrib/blob/a804ca34a9f9711e22cf0e720c09edf58b4615fb/etc/findbugs.xml#L656
So can you please report it on [fb-contrib](https://github.com/mebigfatguy/fb-contrib/issues)?

**Comment by vsTianhao** [comment]
thanks

**Comment by JuditKnoll** [comment]
Closing this as the issue got reported at fb-contrib: https://github.com/mebigfatguy/fb-contrib/issues/474
